modules/rosa-hcp-upgrade-options.adoc

```diff
@@ -0,0 +1,39 @@
+:_mod-docs-content-type: CONCEPT
+[id="rosa-upgrade-options_{context}"]
+= Upgrade options for {hcp-title} clusters
+
+In OpenShift, upgrading means provisioning a new component with updated software and using it to replace an existing component that has outdated software.
+
+You can control the impact of upgrades to your workload by controlling which parts of the cluster are upgraded, for example:
+
+Upgrade only the hosted control plane:: This does not impact your worker nodes.
+
+Upgrade nodes in a single machine pool:: This initiates a rolling replacement of nodes in the specified machine pool, and temporarily impacts the worker nodes on that machine pool. This does not impact nodes on other machine pools in the cluster.
+
+Upgrade nodes in multiple machine pools simultaneously:: This initiates a rolling replacement of nodes in the specified machine pools, and temporarily impacts the worker nodes on those machine pools. You can run this type of upgrade as a single command, or as multiple commands.
+
+Upgrade the hosted control plane, followed by nodes in one or more machine pools:: This initiates upgrade of the hosted control plane, followed by a rolling replacement of nodes in the specified machine pools.
++
+[IMPORTANT]
+====
+These upgrades happen in sequence because the hosted control plane and nodes in machine pools cannot be upgraded simultaneously. When an upgrade of the hosted control plane is in progress, nodes in the machine pools cannot be upgraded. When upgrade is in progress for nodes in the machine pools, the hosted control plane cannot be upgraded.
+====
+
```

Comment:
Does there need to be a + here and following this important note to keep the notes and text in line?

Reply:
Only on the first admonition, although it does look a bit weird to have them not aligned one after another... might rework this a bit

Reply:
Made the content of the first admonition just part of the bullet.